quickwit-search/src/leaf.rs

```diff
@@ -68,15 +68,23 @@ async fn warmup(
     quickwit_collector: &QuickwitCollector,
 ) -> anyhow::Result<()> {
     warm_up_terms(searcher, query).await?;
-    warm_up_fastfield(searcher, quickwit_collector).await?;
+    warm_up_fastfields(searcher, quickwit_collector).await?;
     Ok(())
 }
 
-async fn warm_up_fastfield(
-    _searcher: &Searcher,
-    _quickwit_collector: &QuickwitCollector,
+async fn warm_up_fastfields(
+    searcher: &Searcher,
+    quickwit_collector: &QuickwitCollector,
 ) -> anyhow::Result<()> {
-    // TODO implement me
+    let mut warm_up_futures = Vec::new();
+    for field in quickwit_collector.fast_fields() {
+        for segment_reader in searcher.segment_readers() {
+            let fast_field_slice = segment_reader.fast_fields().fast_field_data(field, 0)?;
```

Comment:
we also need to warm up the timestamp fielld if there is filtering involved

Reply:
Not sure i understand this, but the `timestamp_field` is already part of the fast fields list as long as there is one.

Reply:
can you also check that the field is fast just after making the resolving ?